.gitlab/common/lib.sh

```diff
@@ -193,3 +193,74 @@ check_bootnode(){
     echo "    Bootnode appears unreachable"
     return 1
 }
+
```

Comment:
Those changes should not land under `.gitlab/common` but `scripts/common` or arguably `.github/scripts/common/`

Reply:
I actually think `scripts/common` would be appropriate for anything we may want to use as well on the Gitlab side.
Both CI could benefit from common scripts and helpers, if we split everything into github/gitlab, there will be duplication that would be better avoided.

Reply:
Currently we don't have common scripts between github and gitlab CI (there is `.github/common/lib.sh` but it's not used and should be removed). I'd prefer to put files related to different CIs to their folder. If in the future we have a lot of common scripts then we could think of merging them to some common folder.

Reply:
Ok, I'll move it under the `./gihtub`  folder for now

Reply:
addressed